### ISSUE TITLE
Daily Consumption Charge

### ISSUE THREAD

**Issue by jaysbeekay** [gurbyz/power-wheel-card#58]
In relation to the money view, it would be really useful to have the ability to specifify a daily consumption charge (flat rate) e.g. My power company charges me $0.94 per day to have the power connected so being able to include this within the Power Wheel card allows me to work out the total cost of power per day.

Thanks

**Comment by staticlama** [comment]
Agree, this would be a handy feature to get an idea of true electricity costs. At the moment I get this from pvoutput.
